sourcegraph/sourcegraph
# New Cloud CTA Section on /search

This PR closes #43253 and adds a new CTA Section on `/search`.

## Test plan
- Check that the CTA section is gracefully responsive down to mobile (no mobile design was provided so there's no requirements, but just ensure it looks nice)
- Check that all CTA links work and go to their corresponding locations (signup.sg, docs.sg)
- Check that event data is being logged for the top nav cta, and both links in the Cloud CTA Section (signup.sg, docs.sg)

## Follow Ups
- [x] Logging CTA events on signup.sourcegraph.com as defined in the Metrics artboard will be created in #43320 
- [ ] After the page is live, I will complete the configuration of [the A/B test for positioning](https://optimize.google.com/optimize/home/?authuser=0&hl=en-US#/accounts/4705180113/containers/15523203/experiments/9) the section below the hero. The base metric position will start with this section being placed above the hero, as I anticipate it performing much better.

## App preview:

- [Web](https://sg-web-brett-cloud-cta.onrender.com/search)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
